transforms/csdl/preprocess_csdl_test_output.xml

```diff
@@ -562,11 +562,11 @@
             </Collection>
           </Annotation>
         </Property>
-      </EntityType>	  
-	  <EntityType Name="workbookFunctionResult" BaseType="graph.entity">
-		<Property Name="error" Type="Edm.String" />
-		<Property Name="value" Type="Edm.Untyped" />
-	  </EntityType>
+      </EntityType>
+      <EntityType Name="workbookFunctionResult" BaseType="graph.entity">
+        <Property Name="error" Type="Edm.String" />
+        <Property Name="value" Type="graph.Json" />
```

Comment:
@timayabi2020, is there a specific reason we're switching from `Edm.Untyped` to `graph.Json`? A transformation was added to convert all instances of `graph.Json` to `Edm.Untyped`, so it will eventually revert to `Edm.Untyped` anyway.

https://github.com/microsoftgraph/msgraph-metadata/pull/628

Reply:
That was what is defined in the test input file and when you run the transform.ps1 script, it automatically changes the parameter types. Should the input file be changed as well?